WINDUPRULE-410 camel-rxjava2 renamed

johnpoth opened the pull request:
https://issues.jboss.org/browse/WINDUPRULE-410

Tested executing $ mvn -Dtest=WindupRulesMultipleTests -DrunTestsMatching=xml-renamed-components clean surefire-report:report

windupgithubbot1 commented:
Can one of the admins verify this patch?

mrizzi commented:
ok to test

johnpoth commented:
It has conflicts with https://github.com/windup/windup-rulesets/pull/386

I propose to close this until the above PR has been merged. WDYT @mrizzi  ?

mrizzi commented:
Hi @johnpoth, can you merge latest master into this PR and solve the conflicts please?
The time to start the reviews has come :+1:

johnpoth commented:
@mrizzi updated, thanks for reviewing !

mrizzi commented on rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml:
```diff
@@ -28,6 +28,22 @@
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-rxjava2` artifact has been renamed" effort="1" category-id="mandatory" >
+                    <message>`org.apache.camel:camel-rxjava2` artifact has been renamed in Apache Camel 3 to `org.apache.camel:camel-rxjava` artifact</message>
```
```suggestion
                    <message>`org.apache.camel:camel-rxjava2` artifact has been renamed in Apache Camel 3 to `org.apache.camel:camel-rxjava` artifact</message>
                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_renamed_components" title="Camel 3 - Migration Guide: Renamed components" />
```

johnpoth commented:
Hi @mrizzi good catch! Indeed the website was wrong: it's the camel-rx component that was removed. I just updated the website.

I have also added a link as you did in https://github.com/windup/windup-rulesets/pull/393, thanks !
